test-integration/helpers/course-select.coffee

```diff
@@ -1,17 +1,37 @@
 selenium = require 'selenium-webdriver'
 {TestHelper} = require './test-element'
 
+
+COMMON_ELEMENTS =
+  courseLink: (appearance, isCoach = false) ->
+    dataAttr = 'data-appearance'
+
+    if appearance?
+      dataAttr += "='#{appearance}'"
+
+    if isCoach
+      teacherLink = "[#{dataAttr}] > [href*='cc-dashboard']"
```

Comment:
Could we add classes to the elements so it is easier to find in the DOM? It seems like they'd be less fragile too